indico/htdocs/js/indico/modules/sessions.js

```diff
@@ -0,0 +1,121 @@
+(function(global) {
+    'use strict';
+
+    function setupTableSorter() {
+        $('.sessions .tablesorter').tablesorter({
+            cssAsc: 'header-sort-asc',
+            cssDesc: 'header-sort-desc',
+            cssInfoBlock: 'avoid-sort',
+            cssChildRow: 'session-blocks-row',
+            headerTemplate: '',
+            sortList: [[1, 0]]
+        });
+    }
+
+    function setupPalettePickers() {
+        $('.palette-picker-trigger').each(function() {
+            var $this = $(this);
+            $this.palettepicker({
+                availableColors: $this.data('colors'),
+                onSelect: function(background, text) {
```

Comment:
It would be nice if `onSelect` could return a Deferred object. In that case you could revert to the previous color in case of an error (but it's very much just "nice-to-have" so nothing that needs to be done now)